packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1655,10 +1663,11 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       _stopCursorTimer(resetCharTicks: false);
       _startCursorTimer();
     }
+    _updateEditingValueInProgress = false;
   }
 
-  bool _isSelectionOnlyChange(TextEditingValue value) {
-    return value.text == _value.text && value.composing == _value.composing;
+  bool _isSelectionOnlyChanged(TextEditingValue value) {
+    return value.text == _value.text && value.composing == _value.composing && value.selection != _value.selection;
```

Comment:
I'm not sure why the selection wasn't checked here in the first place.  This seems to make more sense.

Reply:
Do we still need this, with the changes made in line 1650 through 1656 (i.e., the additional case handling when `_value == value` )?

Reply:
@LongCatIsLooong 
Yes, you are correct. But the name of this function is ambiguous, or change the name instead.

Reply:
It appears the method is only used here. You can get rid of it and use the expression instead (and maybe leave a comment that says if the change is selection only)

Reply:
Good idea.
Done.